src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,66 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @OneToOne
+    private Movie movie;
+
+    @OneToOne
+    private CinemaHall cinemaHall;
```

Comment:
Can be multiple movie sessions per movie and multiple movie sessions in a cinema hall